plugins/processors/tag_limit/README.md

```diff
@@ -0,0 +1,27 @@
+# Tag Limit Processor Plugin
+
+Use the `tag_limit` processor to ensure that only a certain number of tags are
+preserved for any given metric, and to choose the tags to preserve when the
+number of tags appended by the data source is over the limit.
+
+This can be useful when dealing with output systems (e.g. Stackdriver) that
+impose hard limits on the number of tags/labels per metric or where high
+levels of cardinality are computationally and/or financially expensive.
+
+### Configuration
+
+```toml
+[[processors.tag_limit]]
+  ## Maximum number of tags to preserve
+  limit = 3
+
+  ## List of tags to preferentially preserve
+  keep = ["environment", "region"]
```

Comment:
What do you think about calling this `include`? An argument for doing so would be that it would simplify adding in an `exclude` in the future if desired.

Reply:
I went back and forth on that and eventually decided that `include` generally indicates an exclusive inclusion, whereas this code will allow unlisted tags through as along as we're still under the limit.

Reply:
(feel free to unresolve if you disagree)